lib/validators/liveValidator.ts

```diff
@@ -667,6 +667,7 @@ export class LiveValidator {
         apiVersion = C.unknownApiVersion
       }
       providerNamespace = providerNamespace.toLowerCase()
+      apiVersion = apiVersion.toLowerCase()
       queryStr = parsedUrl.query
```

Comment:
Can we make queryStr lowercase too for consistency?

Reply:
Sure

Reply:
Sure, I'll make the change. It seems like in previous code block the function converts all keys to lowercase but not values

Reply:
Updated